pwiz_tools/Skyline/TestFunctional/FullScanGraphTest.cs

```diff
@@ -334,7 +338,7 @@ private void TestIonMatchToleranceUnitSetting()
             });
             WaitForGraphs();
             var graphLabels = SkylineWindow.GraphFullScan.IonLabels;
-            Assert.AreEqual(Skyline.Program.SkylineOffscreen ? 49 : 1, graphLabels.Count());
+            Assert.AreEqual(Skyline.Program.SkylineOffscreen ? 48 : 1, graphLabels.Count());

```

Comment:
This number has changed because calling the settings dialogue resets the special ion list and there was one reporter ion in this spectrum.